client.go

```diff
@@ -878,6 +902,20 @@ func (c *Client) Start() (addr net.Addr, err error) {
 				return nil, fmt.Errorf("error parsing server cert: %s", err)
 			}
 		}
+
+		if c.config.GRPCBrokerMultiplex && c.protocol == ProtocolGRPC {
+			if len(parts) <= 6 {
+				return nil, errors.New("client requested gRPC broker multiplexing but plugin does not " +
+					"support the feature; for Go plugins, you will need to update the github.com/hashicorp/go-plugin" +
+					"dependency and recompile")
+			}
+			if muxSupported, err := strconv.ParseBool(parts[6]); err != nil {
+				return nil, fmt.Errorf("error parsing %q as a boolean for gRPC broker multiplexing support", parts[6])
+			} else if !muxSupported {
+				return nil, errors.New("client requested gRPC broker multiplexing but plugin does not " +
```

Comment:
Suggestion: put this error as a global variable so that clients can more easily gracefully fallback to setting mux to `false`.

Reply:
Good idea - I used it here and above as well, and also added a test to check we receive the expected error.